docs/src/app/(public)/blog/[...slug]/page.tsx

```diff
@@ -87,3 +88,34 @@ export async function generateStaticParams() {
     slug: slug.split('/'),
   }));
 }
+
+export async function generateMetadata(
+  { params }: BlogProps,
+  parent: ResolvingMetadata
+): Promise<Metadata> {
+  const slugPath = params.slug;
+  const slug = slugPath.join('/');
+
+  const page = await reader.collections.blog.read(slug);
+
+  const parentOGImages = (await parent).openGraph?.images || [];
+  const parentTwitterSite = (await parent).twitter?.site ?? '';
+
+  return {
+    title: page?.title,
+    description: page?.summary,
+    openGraph: {
+      title: page?.title,
+      description: page?.summary,
+      url: `https://keystatic.com/blog/${slug}`,
+      type: 'article',
+      images: ['', ...parentOGImages],
```

Comment:
What does the `''` do here?

Reply:
Ah it does nothing I think, I left it there because I plan to add a custom OG image for each blog post and this is where I'd put that in. Is it better to remove that for now, and just do `images: parentOGImages,` you reckon?

Reply:
Yeah, I think capture the intention in a ticket and we can come back to it. And make this `images: parentOGImages` as you suggest 👍

Reply:
That makes total sense, will do!